pkg/controller/plan/migration.go

```diff
@@ -1200,6 +1201,53 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 	return
 }
 
+// Load host CRs.
+func (r *Migration) loadHosts() (hosts map[string]*v1beta1.Host, err error) {
```

Comment:
move to kubevirt?

Reply:
it's unrelated to kubevirt but to the source provider, but I don't mind moving it